<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListEditForm</class>
 <widget class="QWidget" name="ListEditForm">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>431</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>431</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QLineEdit" name="keywordInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>400</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>添加关键词</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>关键词列表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>400</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>保存列表</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>创建者</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>列表简介</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="listAuthorInput">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="listIntroInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>列表名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="listTitleInput">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="keywordsTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>381</width>
     <height>192</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>192</horstretch>
     <verstretch>200</verstretch>
    </sizepolicy>
   </property>
   <property name="autoScroll">
    <bool>false</bool>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="importButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>从B站导出的XML或DPLIST列表文件导入</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
